commands/serve.js

```diff
@@ -15,7 +17,8 @@ const handler = (argv = {}) => {
     {
       config,
       on: {
-        'build-started': () => console.log('\nBuild started...\n'),
+        'build-started': () =>
+          console.log(chalk.bgBlue.white(`\nBuild started...\nWebpack server running at ${chalk.underline(host)}\n`)),
```

Comment:
it ill cause a conflict with @piotr-gawlowski PR, i ill resolve it after #39 merge